gp-conditional-pricing/gpcp-fuzzy-comparison.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * Gravity Perks // GP Conditional Pricing // Fuzzy Comparison for operator "is"
+ * http://gravitywiz.com/documentation/gravity-forms-conditional-pricing/
+ *
+ * This snippet adds fuzzy comparison for "is" operator with the GPCP rules.
+ *
+ */
```

Comment:
Update header to match the fact that this applies to all GF conditional logic rules.